docs/github-secrets.md

```diff
@@ -0,0 +1,25 @@
+Configuring Github Application Secrets
+===
+
+The process of creating a GitHub application is explained in
+[Pipelines-as-Code documentation](https://pipelinesascode.com/docs/install/github_apps/#manual-setup).
+The same secret described there, should be deployed to the `build-service` and
+`integration-service` namespaces.
+
+To do that, repeat the `kubectl create secret` command described there for the
+`pipelines-as-code` namespace also to those namespace:
+
+```bash
+kubectl -n pipelines-as-code create secret generic pipelines-as-code-secret \
```

Comment:
I know it is described in the PaC documentation, but to make it easy, we can add it here (the same goes for all the coomands)
```suggestion
kubectl -n pipelines-as-code create secret generic pipelines-as-code-secret \
        --from-literal github-private-key=<GH_APP_PRIVATE_KEY> \
        --from-literal github-application-id=<GH_APP_ID> \
```

Reply:
If we're repeating stuff that is written there, it means we can potentially go out of sync when that changes. Here we need this command 4 times, so I'm worried about repeating it...

Anyhow, this is not a change we introduce in this PR -- we just move it to a separate file, so I prefer not to change it as part of this PR.